server/GameServer.cpp

```diff
@@ -370,58 +369,61 @@ void GameServer::CheckTimedUpdates()
 		auto timeDiff = currentTime - gameSessions[i].time;
 		
 		// Check if no updates have occured for a while
-		if (timeDiff < InitialReplyTime)
+		if (timeDiff >= InitialReplyTime)
 		{
-			continue;
+			DoTimedUpdate(i, timeDiff);
 		}

```

Comment:
Uhh, similar concern here. Inverting the logic while extracting a function at the same time produces a rather noisy and hard to verify diff. My eyes are glazing over on this one.